### PR TITLE
Add note about mongosync perf issue when Balancer is enabled

### PR DESCRIPTION
**Pull request by MongoCaleb**
## Notes
This re-instates a recently removed warning and updates it with only the pertinent information regarding copy performance issues.

## JIRA
https://jira.mongodb.org/browse/DOCSP-35315

## Build Log
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65b190a49a69c5a89fc525c1

## Staging
https://preview-mongodbmongocaleb.gatsbyjs.io/cluster-sync/DOCSP-35315/multiple-mongosyncs/#start-multiple-mongosync-instances

### CODE REVIEW

File: source/includes/sharding-warning.rst [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-
-   To avoid this, it is recommended that you run the :dbcommand:`balancerStop`
-   command on the destination cluster before starting a sync.  Once ``mongosync``
-   completes the sync, you can restart the balancer with the 
-   :dbcommand:`balancerStart` command.
-
+.. important::
+
+   If the destination is a sharded cluster with the balancer enabled, write 
+   performance of the destination cluster may be degraded during ``mongosync``'s
```
Comment (ltran-mdb2) [inline source/includes/sharding-warning.rst:5]: [style guide] Per the writing guide,  could we make this [active voice](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)? 

For ex, "write performance of the destination cluster may decline during ..."

File: source/includes/sharding-warning.rst [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
+.. important::
+
+   If the destination is a sharded cluster with the balancer enabled, write 
+   performance of the destination cluster may be degraded during ``mongosync``'s 
+   Collection Copy phase. To warn of this potential performance impact,   
+   ``mongosync`` runs the :method:`sh.getBalancerState()` command during 
+   initialization. If it returns ``true``, it logs a warning. 
+   
+   Consider disabling the balancer on the destination cluster if write performance
```
Comment (ltran-mdb2) [inline source/includes/sharding-warning.rst:10]: [style guide]: Could we reword this to [put the conditional clause first?](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure)

File: source/includes/sharding-warning.rst [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-   completes the sync, you can restart the balancer with the 
-   :dbcommand:`balancerStart` command.
-
+.. important::
+
+   If the destination is a sharded cluster with the balancer enabled, write 
+   performance of the destination cluster may be degraded during ``mongosync``'s 
+   Collection Copy phase. To warn of this potential performance impact,   
+   ``mongosync`` runs the :method:`sh.getBalancerState()` command during 
+   initialization. If it returns ``true``, it logs a warning.
```
Comment (ltran-mdb2) [inline source/includes/sharding-warning.rst:8]: [style guide] Would it be possible to [clarify the antecedents](https://www.mongodb.com/docs/meta/style-guide/writing/use-pronouns-carefully/) here? It may be unclear to readers what the first and second "it" in the last sentence refers to.

**Review by ltran-mdb2 — COMMENTED** [review]
Thanks for picking this up @MongoCaleb ! I left a few comments for consideration

File: source/includes/sharding-warning.rst (hunk 1 above)
Comment (MongoCaleb) [reply source/includes/sharding-warning.rst:5]: 👍  but kept "degrade" over "decline".

**Review by ltran-mdb2 — APPROVED** [review]
LGTM! Thanks Caleb!

File: source/includes/sharding-perf.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+If the destination is a sharded cluster with the balancer enabled, write 
+performance of the destination cluster may degrade during ``mongosync``'s 
+Collection Copy phase. To warn of this potential performance impact,   
+``mongosync`` runs the :method:`sh.getBalancerState()` command during 
+initialization. If ``getBalancerState()`` returns ``true``, ``mongosync`` 
+logs a warning. 
+
+If write performance during migration is impacted, consider disabling the 
+balancer on the destination cluster . You stop the balancer with the
```
Comment (jeff-allen-mongo) [inline source/includes/sharding-perf.rst:10]: There is an extra space before this period.

```suggestion
balancer on the destination cluster. You stop the balancer with the 
```

**Review by jeff-allen-mongo — APPROVED** [review]
